funnel/forms/space.py

```diff
@@ -74,3 +74,13 @@ def validate_bg_color(self, field):
 
 class RsvpForm(forms.Form):
     status = forms.RadioField("Status", choices=[(k, RSVP_STATUS[k].title) for k in RSVP_STATUS.USER_CHOICES])
+
+
+class ParticipantForm(forms.Form):
+    fullname = forms.StringField(__("Full Name"), validators=[forms.validators.DataRequired()])
+    email = forms.EmailField(__("Email"), validators=[forms.validators.DataRequired(), forms.validators.Length(max=80)])
+    phone = forms.StringField(__("Phone number"), validators=[forms.validators.DataRequired(), forms.validators.Length(max=80)])
+    city = forms.StringField(__("City"), validators=[forms.validators.Length(max=80)])
+    company = forms.StringField(__("Company"), validators=[forms.validators.DataRequired()])
+    job_title = forms.StringField(__("Job Title"), validators=[])
```

Comment:
The `validators` parameter is not required when empty, but in this case it needs a `Length` validator.